snippets/halium.xml

```diff
@@ -10,7 +10,7 @@
       <linkfile src="." dest="halium/devices" />
   </project>
   <project path="vendor/halium/audioflingerglue" name="Halium/audioflingerglue" revision="master" />
-  <project path="vendor/halium/biometryd" name="ubports/core/biometryd" revision="ubports/xenial_-_android9" remote="gitlab" />
+  <project path="vendor/halium/biometryd" name="ubports/development/core/biometryd" revision="ubports/xenial_-_android9" remote="gitlab" />
```

Comment:
Now the branch should be `main` for 20.04 and similar changes should also be reflected in `platform-api` repo below.

#80 probably should also be updated along with Halium 9 and 12 getting PRs (should I submit those?) to do similar changes and hopefully merge all around the same time as they'll likely need manual intervention to fix `repo sync` after URL/branch change.